### PR TITLE
LPS-82464 Add unit tests

### PR DESCRIPTION
**Pull request by pedroqueiroz**

### CODE REVIEW

**Comment by pedroqueiroz** [comment]
ci:test:sf

**Comment by pedroqueiroz** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v2_0_3/UpgradeDataProviderInstance.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.util.StringBundler;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Inácio Nery
```
Comment (rafaprax) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v2_0_3/UpgradeDataProviderInstance.java]: Hey @inacionery , thanks for collaboration. 😄 JK.

@pedroqueiroz fix author. Thx

**Comment by rafaprax** [comment]
ci:test:relevant

**Comment by pedroqueiroz** [comment]
ci:stop

**Pull request closed without merging**